packer/install-dependencies.sh

```diff
@@ -448,6 +423,28 @@ for KUBERNETES_VERSION in ${K8S_VERSIONS}; do
   fi
 done
 
+# pull patched hyperkube image for AKS
+# this is used by kube-proxy
+PATCHED_HYPERKUBE_IMAGES="
+v1.12.8_v0.0.5
+v1.13.10_v0.0.5
+v1.13.11_v0.0.5
```

Comment:
We should probably delete the duplicate versions in the `K8S_VERSIONS` array above, right? Otherwise the VHD won't have enough storage space (limited to 30 GB of local storage). Each of these additional hyperkube images is roughly 1 GB in size.

Reply:
This spec will not work with duplicate images, we don't have enough storage. Can't build a VHD from this spec. Every duplicated image in `K8S_VERSIONS` above that has an equivalent here needs to be removed to free up space.